### DISCUSSION TITLE
Cross-platform problem

### DISCUSSION THREAD

**Discussion by Nitestack** [twpayne/chezmoi#3436] [Q&A] [answered]
So I have a Neovim configuration. 
For Windows, the directory is `$env:LOCALAPPDATA\nvim` (`C:\Users\<user>\AppData\Local\nvim`)
For Unix operating systems, it's `~/.config/nvim`.

How can I maintain one single Neovim configuration folder for both operating systems?

Thanks in advance!

**Comment by bradenhilton** [comment]
https://www.chezmoi.io/user-guide/manage-machine-to-machine-differences/#handle-different-file-locations-on-different-systems-with-the-same-contents

**Reply by Nitestack** [reply]
I took a look at this. To my understanding (I could be wrong, so correct me here please) only files could be added to `.chezmoitemplates\` (which is for sharing files) directory. I could add the whole folder, of course, but then I have to create a templ file for every file in my Neovim configuration for both platforms.

**Reply by twpayne** [reply]
The long term plan to address this is https://github.com/twpayne/chezmoi/issues/2273, but this will require significant work to implement, so there's no expected date for it yet.

**Reply by Nitestack** [reply]
Oh, thanks for the answer. What do you suggest for now doing? Symlinking?

**Reply by twpayne** [reply, marked as answer]
Symlinking is the easiest method at the moment. It will be trivial to change it to target path rewriting in the future when that feature lands.

**Reply by Nitestack** [reply]
Alright, thanks, I'm looking forward to that feature! 

wanted to say, this is an awesome project you guys built! Very easy to bootstrap a new environment. Keep up the good work
